packages/apollo-server-core/src/graphqlOptions.ts

```diff
@@ -69,6 +69,14 @@ export type DataSources<TContext> = {
 
 export interface PersistedQueryOptions {
   cache: KeyValueCache;
+  /**
+   * Specified in **seconds**, this time-to-live (TTL) value limits the lifespan
+   * of how long the persisted query should be cached.  To specify a desired
+   * lifespan of "infinite", set this to `null`, in which case the eviction will
+   * be determined by the cache's eviction policy, but the record will never
+   * simply expire.
+   */
+  ttlSeconds?: number | null;
```

Comment:
@trevor-scheer Can you try to convince me to change this parameter to be `ttl`, rather than `ttlSeconds`?

Considerations:

* Against: It's come up a number of times before that parameters that define units of time could be more clear by specifying their precision as part of the name.  (This is the only reason I actually put `Seconds` in the name.)
* For: The existing interfaces which this translate to are still `ttl`, for better or for worse, but that's what they are.
* For: It's not "seconds" when set to `null`, so `ttlSeconds: null` is a bit peculiar.
  * Along the same lines, this PR changes those existing interfaces to also support `null`.
* For: While including the precision makes sense in many cases, TTLs are _very often_ (e.g. Memcached, Redis, HTTP `Cache-control`, and so on).  On the other hand, [`lru-cache`](https://npm.im/lru-cache)  sets its `maxAge` in milliseconds.  To be honest, I find such flexibility unnecessary.  Is it truly wise to cache things with sub-second precision?
* For: Thanks to the TSDoc annotation provided directly above this, users with editors that support it are automatic given IntelliSense clarity that this in seconds.

Reply:
Of course, anyone should feel free to chime in on this naming puzzle.